app/views/layouts/_admin_sidebar_index.html.haml

```diff
@@ -32,3 +32,7 @@
       = link_to(admin_revision_history_path) do
         %span.fa.fa-history
         Revision History
+    %li
+      = link_to(admin_organizations_path) do
+        %span.fa.fa-group
+        Organizations
```

Comment:
The idea is to merge the PRs as soon as they are ready. If we just create a huge PR at the end of GSoC, it will be impossible to merge.